hazelcast-client/src/test/java/com/hazelcast/client/test/TestClientRegistry.java

```diff
@@ -236,7 +236,7 @@ public void run() {
 
                 @Override
                 public String toString() {
-                    return "Runnable message " + clientMessage;
+                    return "Runnable message " + clientMessage + ",localAddress " + localAddress + "," + MockedClientConnection.this;
```

Comment:
Why don't you just implement `toString()`  in `MockedClientConnection` class? In that case, you would write;

```java
return "Runnable message " + clientMessage + ", Connection: " + MockedClientConnection.this;
```